src/Evolution/Systems/Cce/Actions/InitializeCharacteristicEvolutionScri.hpp

```diff
@@ -39,10 +45,16 @@ namespace Actions {
  * mechanism, so `Actions::SetupDataBox` must be present in the `Initialization`
  * phase action list prior to this action.
  */
-template <typename ScriValuesToObserve>
+template <typename ScriValuesToObserve, typename BoundaryComponent>
 struct InitializeCharacteristicEvolutionScri {
-  using initialization_tags =
-      tmpl::list<InitializationTags::ScriInterpolationOrder>;
+  using initialization_tags = tmpl::flatten<tmpl::list<
+      InitializationTags::ScriInterpolationOrder,
+      tmpl::conditional_t<
+          tt::is_a_v<AnalyticWorldtubeBoundary, BoundaryComponent>,
+          tmpl::list<Tags::AnalyticBoundaryDataManager>, tmpl::list<>>>>;
+  using initialization_tags_to_keep = tmpl::conditional_t<
+      tt::is_a_v<AnalyticWorldtubeBoundary, BoundaryComponent>,
+      tmpl::list<Tags::AnalyticBoundaryDataManager>, tmpl::list<>>;
```

Comment:
Because they are only used by a single component, so don't need to clutter the databox of every component.

Reply:
I see. If you want to move this logic to the parallel component as you did for `InitializeFirstHypersurface` then you could add this initialization tag in a separate action and have the parallel component decide whether to add it to the PDAL or not. Or just add it to the global cache along with `AnalyticInitializeJ`, just to keep things simple 🤷‍♂️

Reply:
It's also ill-suited to be in the global cache because it some analytic solutions need to modify internal state during their evolution ('analytic' here is used roughly -- the one in question actually numerically integrates a restricted set of variables).
I disagree that adding another action just to add the one tag is any simpler -- the analytic solution in this case is used during the scri observation action (because it needs to output the news from the analytic solution), so I think this initialization action is the right place for it.